resolver.go

```diff
@@ -240,7 +240,7 @@ func (r *Resolver) exchange(ctx context.Context, host, qname, qtype string, dept
 func (r *Resolver) exchangeIP(ctx context.Context, host, ip, qname, qtype string, depth int) (RRs, error) {
 	dtype := dns.StringToType[qtype]
 	if dtype == 0 {
-		dtype = dns.TypeA
```

Comment:
This doesn't do weird things to the output when using `-w`?

Reply:
I could try reverting this.

Reply:
Done. Tests pass. Will merge.